static/js/rest.js

```diff
@@ -102,13 +107,19 @@ export const endpoints: Array<Endpoint> = [
   {
     name: 'payment',
     url: '/api/v0/payment/',
-    makeOptions: (total, klassId) => ({
+    fetchOptions: (params) => ({
```

Comment:
ok, I think I see where you're coming from here. What I did in MM was just have the generated action take a rest parameter (i.e. a function argument of the form `...params`) and pass `...params` into all of the relevant functions, so the `fetchOptions`, the url generating function, etc. I think your approach will work equally well though!